x/ibc/keeper/querier.go

```diff
@@ -33,12 +32,7 @@ func NewQuerier(k Keeper, legacyQuerierCdc codec.JSONMarshaler) sdk.Querier {
 		case connectiontypes.SubModuleName:
 			err = sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unknown IBC %s query endpoint", connectiontypes.SubModuleName)
 		case channeltypes.SubModuleName:
-			switch path[1] {
-			case channeltypes.QueryChannelClientState:
-				res, err = channelkeeper.QuerierChannelClientState(ctx, req, k.ChannelKeeper, legacyQuerierCdc)
-			default:
-				err = sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unknown IBC %s query endpoint", channeltypes.SubModuleName)
-			}
+			err = sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "unknown IBC %s query endpoint", channeltypes.SubModuleName)
```

Comment:
ditto for other submodules
```suggestion
			err = sdkerrors.Wrapf(sdkerrors.ErrUnknownRequest, "%s query not supported. Use gRPC instead", channeltypes.SubModuleName)
```

Reply:
I'll update this in the last pr to just return a single error that IBC does not support legacy queries and to use gRPC instead.